src/renderer/util/contextMenu.js

```diff
@@ -61,6 +61,30 @@ export function openContextMenu(event, templates = []) {
   injectDevelopmentTemplate(event, templates);
   remote.Menu.buildFromTemplate(templates).popup();
 }
+
+export function openEditorMenu(event, codeMirror) {
```

Comment:
It might be helpful to add a comment above this function describing its use for the markdown editor